modules/ROOT/pages/policies-policy-categories.adoc

```diff
@@ -0,0 +1,40 @@
+= Policy Categories
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the Default policy name, its category and the function that it performs. +
+You can apply only one policy with a particular `providedCharacteristics` at a given time in an API.
```

Comment:
This does not seem to apply to the listing and categorizing of policies but more with the application and use of them. Please move to the appropriate location, one in which the audience is configuring policies.

Reply:
Done! Thanks!